io/src/main/java/org/commonjava/maven/ext/io/SettingsIO.java

```diff
@@ -58,7 +58,7 @@ public void write( Settings settings, File settingsFile )
     {
         try
         {
-            PrintWriter printWriter = new PrintWriter( settingsFile, StandardCharsets.UTF_8.toString() );
+            PrintWriter printWriter = new PrintWriter( settingsFile, settings.getModelEncoding() );
```

Comment:
Why this change please?

Reply:
Ah are you trying to get hold of 
```<?xml version="1.0" encoding="utf-8"?>``` ?

Reply:
Yes, since we're writing out the settings file, we should want to keep the encoding of the original XML file, so I am trying to use `settings.getModelEncoding()` API to return the original encoding (which also should safely return `"UTF-8"` by default anyway as opposed to `null` anyway).

Reply:
Ok cool